packages/compadre/examples/GMLS_Manifold_Multiple_Evaluation_Sites.cpp

```diff
@@ -644,7 +552,7 @@ Kokkos::initialize(argc, args);
         // get tangent vector and see if orthgonal to coordinate (it should be on a sphere)
         for (int j=0; j<dimension-1; ++j) {
             double tangent_inner_prod = 0;
-            for (int k=0; k<dimension; ++k) {
+            for (int k=0; k<std::min(dimension,3); ++k) {
```

Comment:
Why don't you error out if dimension > 3 ?

Reply:
We can error out earlier in the code for dimension. I suppose that check would be put in the CommandLineParser class. Would that solve this issue though? The problem here is that 7.2.0 gcc has the flag for aggressive loop reduction with warnings as errors.  The contents in the loop can't be reduced if dimension can exceed 3, so this this a way of telling the compiler it won't exceed 3.